cypress/integration/login.js

```diff
@@ -1,92 +1,116 @@
-describe('Sourcebooks testLogIn', function() {
-    
-    var roles = [
-        { name: 'User', 'tabCount': 1 }, 
-        { name: 'Team Lead', tabCount: 2 }, 
-        { name: 'Manager', tabCount: 5 }, 
-        { name: 'Accountant', tabCount: 5}, 
-        { name: 'Admin', tabCount: 6 }
-    ];
+import LoginPage from '../obj/loginPage';
+import ClientsPage from '../obj/clients/clientsPage';
+import ClientCreatePage from '../obj/clients/clientCreatePage';
+import TasksPage from '../obj/tasks/tasksPage';
+import TaskCreatePage from '../obj/tasks/taskCreatePage';
+import TasksEditPage from '../obj/tasks/taskEditPage';

```

Comment:
I don't think this is used anywhere :)

Reply:
They are used later on :)
in lines 31-38, 77-79, 68-74,  79

Reply:
```
import TasksEditPage from '../obj/tasks/taskEditPage';
import TaskEditPage from '../obj/tasks/taskEditPage';
```
you use `TaskEditPage`, but `TasksEditPage` is not used

Reply:
okey, thank you :)

Reply:
This is still not used anywhere